src/hooks/useDropZone/useDropZone.ts

```diff
@@ -0,0 +1,191 @@
+import * as React from 'react';
+
+export type UseDropZoneAccept = string[];
+
+export interface UseDropZoneParams {
+    accept: UseDropZoneAccept;
+    ref?: React.RefObject<HTMLElement>;
+    disabled?: boolean;
+    onDrop: (items: DataTransferItemList) => void;
+}
+
+const DROP_ZONE_BASE_ATTRIBUTES = {
+    'aria-dropeffect': 'copy' as DataTransfer['dropEffect'],
+    tabIndex: 0,
+    role: 'button',
+};
+
+export interface DroppableProps extends Required<typeof DROP_ZONE_BASE_ATTRIBUTES> {
```

Comment:
We usually name props according to this logic `<ComponentOrHookName> + details`. This is necessary so that all types have their own name space + it improves DX when working with lib types